secant/Features/Profile/ProfileStore.swift

```diff
@@ -115,7 +117,7 @@ extension ProfileReducer {
             SettingsEnvironment(
                 localAuthenticationHandler: .live,
                 mnemonic: environment.mnemonic,
-                SDKSynchronizer: environment.SDKSynchronizer,
+                SDKSynchronizer: LiveWrappedSDKSynchronizer(),
```

Comment:
This is breaking the functionality, `LiveWrappedSDKSynchronizer()` won't be properly initialised + it's potentially breaking all subscribers for the synchronizerStateChanged.